<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>321</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="add_seller">
     <attribute name="title">
      <string>Doctor</string>
     </attribute>
     <widget class="QTextBrowser" name="doctorsList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>431</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Patient</string>
     </attribute>
     <widget class="QTextBrowser" name="patientsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="add">
     <attribute name="title">
      <string>New</string>
     </attribute>
     <widget class="QPushButton" name="newDoctor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New doctor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newPatient">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New patient</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New ticket</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="doctorSurnameInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="doctorNameInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="doctorPatronymicInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="doctorSpecialityInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>surname: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>patronymic:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>speciality:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>surname: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>patronymic:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
